docs/tutorial/kymotracking.rst

```diff
@@ -252,3 +252,85 @@ The candidate with the lowest score is then selected.
 Since this algorithm is specifically looking for curvilinear structures, it can have issues with structures that are
 more blob-like (such as short-lived fluorescent events) or diffusive traces, where the particle moves randomly rather
 than in a uniform direction.
+
+
+Studying diffusion processes
+----------------------------
+
+To study diffusive processes we can make use of the Mean Squared Displacement (MSD).
+There are multiple ways to estimate this quantity.
+We use the following estimator:
+
+.. math::
+
+    \hat{\rho}[n] = \frac{1}{N - n} \sum_{i=1}^{N-n}\left(x_{i+n} - x_{i}\right)^2
+
+where :math:`\hat{\rho}[n]` corresponds to the estimate of the MSD for lag :math:`n`, :math:`N` reflects the number of samples, :math:`n` represents the lag we are computing and the trace positions at time frame i is given by :math:`x_i`.
+
+The reason we use this definition is because it results in the largest number of displacements being used thereby resulting in a well averaged estimate. The downside of this estimator however, is that it suffers from providing very correlated estimates :cite:`qian1991single,michalet2010mean,michalet2012optimal` which needs to be taken into account when doing subsequent analyses.
+
```

Comment:
How about:

> What we can see in this definition is that it uses the same data points several times, thereby resulting in a well averaged estimate.

> However, the downside of this estimator is that the calculated values are highly correlated :cite:`qian1991single,michalet2010mean,michalet2012optimal` which needs to be accounted for in subsequent analyses.

I'm hesitant about calling it an error because there is no error in the MSD estimate itself. The variance just goes up like crazy.

Reply:
sounds good